src/index.js

```diff
@@ -68,14 +70,27 @@ class ToDo {
           }
         }
       });
+
+      checkBox.addEventListener('click', () => {
```

Comment:
- [ ] You can use the change event listener instead of the click event listener on the checkbox. The change event listener will fire when the state of the checkbox changes, regardless of how it was changed (by clicking, using the keyboard, etc.). The click event listener will only fire when the checkbox is clicked by the mouse. Using the change event listener can make your code more robust and accessible. You can learn more about the change event here: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event

I have provided an improved version of your code below. Please review it and consider making the suggested change. If you have any questions or doubts, please let me know.

```.js

      checkBox.addEventListener('change', () => {
        label.classList.toggle('checked');
        if (todo.completed === false) {
          todo.completed = true;
        } else {
          todo.completed = false;
        }
        localStorage.setItem('toDo', JSON.stringify(this.toDo));
      });
```
See project requirements to use change event listener below:
<img width="613" alt="image" src="https://github.com/oluwatobi-fasasi/toDoList/assets/52934128/2357fd12-765e-4c7e-8656-b7d14c09af42">